mof_array/pmf/process_mass_data.py

```diff
@@ -323,4 +322,19 @@ def information_gain(normalize_binned_pmf_results, create_bins_results, experime
 
     return(array_gas_info_gain)
 
-# def choose_best_arrays(information_gain_results):
+def choose_best_arrays(gas_names, experimental_mass_mofs, information_gain_results):
+    """Choose the best MOF arrays by selecting the top KL scores for each gas
+    """
```

Comment:
'choosing arrays' implies you are returning arrays. In this case you are printing them. This is fine, but maybe rename the function "print_best_arrays" or else actually return arrays.